presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -162,17 +162,13 @@ protected void assertQueryFails(@Language("SQL") String sql, @Language("RegExp")
 
     protected void assertQueryFails(Session session, @Language("SQL") String sql, @Language("RegExp") String expectedMessageRegExp)
     {
-        queryRunner.getExclusiveLock().lock();
```

Comment:
The shared lock is acquired inside if the `queryRunner.execute(session, sql)` call below.